core/src/test/scala/org/apache/spark/MapOutputTrackerSuite.scala

```diff
@@ -333,4 +333,63 @@ class MapOutputTrackerSuite extends SparkFunSuite {
     rpcEnv.shutdown()
   }
 
+  ignore("Benchmarking `MapOutputTracker.serializeMapStatuses`") {
```

Comment:
Are there any reasons for it is not implemented as regular benchmark using the `Benchmark` class?

Reply:
I can convert it to a full benchmarking suite using `Benchmark` class. Just want to see the result quickly.